.github/workflows/ci.yml

```diff
@@ -146,6 +146,28 @@ jobs:
     - name: make test
       run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
 
+        #TODO 1. Install Test::V0 to run perl tests
+        #TODO 2. Install tcl to run tcl tests
+  buildtest-gost:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: config
+      run: ./config --strict-warnings && perl configdata.pm --dump
+    - name: make
+      run: make -s -j4
+    - name: checkout gost engine
+      run: git clone --depth 1 -b master https://github.com/gost-engine/engine.git gost-engine-test
```

Comment:
"actions/checkout@v2" is quite useful for this!

``` yaml
    - name: checkout gost engine
      uses: actions/checkout@v2
      with:
        repository: gost-engine/engine
        ref: master
        path: gost-engine-test
```

Reply:
Done. Thanks!

Reply:
Also IMO we should use known-good commit id otherwise we will regress if something is broken in gost-engine master.